### ISSUE TITLE
Output build cmd prompt is not properly displayed in the GUI

### ISSUE THREAD

**Issue by jengee** [blackberry/webworks-gui#214]
Output build information is missing line breaks that exist in the build cmd prompt. Would be more readable if the line breaks were put back in. 

![outputnotformatted](https://cloud.githubusercontent.com/assets/549345/2817903/b90aec26-cede-11e3-9d9f-295d6bab196e.PNG)

**Comment by jsoref** [comment]
@rcruz / @kwallis : i'd like to target this for 2.2. (I'd also like to be able to set targets myself...)

**Comment by jamesjhedley** [comment]
Just wanted to note, this seems to be a Windows issue? I can't seem to reproduce on Mac or Linux. Output on those platforms contain line breaks. Are you seeing the same @jsoref ?

**Comment by jsoref** [comment]
Yes, I suspect it's line endings. We might need to ensure CR-LF...
